DependencyInjection/CompilerPass/DynamicFormFieldTypeCompilerPass.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Sulu\Bundle\FormBundle\DependencyInjection\CompilerPass;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Reference;
+
+/**
+ * Compiler pass for collecting for collecting services tagged with sulu_form.dynamic.type.
+ */
+class DynamicFormFieldTypeCompilerPass implements CompilerPassInterface
```

Comment:
@rs2487 as this service exists only since 1.3 pls update the requirement in the `composer.json`